README.md

```diff
@@ -288,110 +295,15 @@ with dag:
-
-
-@df
-def perform_dataframe_transformation(df: DataFrame):
-    """Train model with Python. You can import any python library you like and treat this as you would a normal
-    dataframe
-    """
-    recent_purchases_dataframe = df.loc[:, "recent_purchase"]
-    return recent_purchases_dataframe
-
-
-@df
-def dataframe_action_to_sql(df: DataFrame):
-    """
-    This function gives us an example of a dataframe function that we intend to put back into SQL. The only thing
-    we need to keep in mind for a SQL return function is that the result has to be a dataframe. Any non-dataframe
-    return will result in an error as there's no way for us to know how to upload the object to SQL.
-    """
-    return df
-
-
-SOURCE_TABLE = "source_finance_table"
-
-s3_path = (
-    f"s3://astronomer-galaxy-stage-dev/thanos/{SOURCE_TABLE}/"
-    "{{ execution_date.year }}/"
-    "{{ execution_date.month }}/"
-    "{{ execution_date.day}}/"
-    f"{SOURCE_TABLE}_"
-    "{{ ts_nodash }}.csv"
-)
-
-with dag:
-    """Structure DAG dependencies.
-    So easy! It's like magic!
-    """
-
-    raw_orders = aql.load_file(
-        path="s3://my/s3/path.csv",
-        file_conn_id="my_s3_conn",
-        output_table=Table(table_name="foo", conn_id="my_postgres_conn"),
-    )
-    agg_orders = aggregate_orders(raw_orders)
-    customers = get_customers()
-    features = join_orders_and_customers(customers, agg_orders)
-    simple_df = perform_dataframe_transformation(df=features)
-    # By defining the output_table int the invocation, we are telling astro where to put the result dataframe
-    dataframe_action_to_sql(
-        simple_df, output_table=Table(table_name="result", conn_id="my_postgres_conn")
-    )
-```
-
-
-## The output_table parameter
+Before we can complete any transformations, we need to define a way to get our tables in and out of Airflow. We can do this by defining either `Table` or `TempTable` objects in the `input_table` and `output_table` parameters of our table instantiations.
 
 ### The Table class
 
-To instantiate a table or bring in a table from a database into the `astro` ecosystem, you can pass a `Table` object into the class. This Table object will contain all necessary metadata to handle table creation between tasks. Once you define it in the beginning of your pipeline, `astro` can automatically pass that metadata along.
+To instantiate a table or bring in a table from a database into the `astro` ecosystem, you can pass a `Table` object into the class. This Table object will contain all of the metadata that's necessary for handling table creation between tasks. Once you define at in the beginning of your pipeline, `astro` can automatically pass that metadata along to downstream tasks.
+
+In the following example, we define the our table in the DAG instantiation. In each subsequent task, we only pass in an input table argument because `astro` automatically passes in the additional context from our original `input_table` parameter.
```

Comment:
```suggestion
In the following example, we define our table in the DAG instantiation. In each subsequent task, we only pass in an input table argument because `astro` automatically passes in the additional context from our original `input_table` parameter.
```